server/src/services/course.service.ts

```diff
@@ -200,6 +200,13 @@ export async function expelMentor(courseId: number, githubId: string) {
   }
 }
 
+export async function restoreMentor(courseId: number, githubId: string) {
```

Comment:
please move to `rsschool-app/server/src/services/mentor.service.ts`

Reply:
Why should this be taken out of the course - this is the same expulsion and restoration of the mentor in the current course?